src/main/java/com/minecolonies/coremod/entity/pathfinding/WalkToProxy.java

```diff
@@ -131,6 +131,7 @@ public boolean walkToBlock(@NotNull BlockPos target, int range, boolean onMove)
                 currentProxy = target;
             }
 
+            //TODO any reason we don't just call proxyList.clear() ?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)